config/manifests/bases/activemq-artemis-operator.clusterserviceversion.yaml

```diff
@@ -2457,6 +2513,7 @@ spec:
   - email: rkieley@redhat.com
     name: Roderick Kieley
   maturity: beta
+  minKubeVersion: 1.25.0
```

Comment:
Can you use a lower version like `1.20.0` unless there is a technical requirement to avoid backward compatibility issues for users running the operator on an outdated Kubernetes cluster?

Reply:
Sure, i will check with adding `1.20.0` and re-run the tests once again.